src/app/process-editor/services/bpmn-js/property-handlers/name.handler.ts

```diff
@@ -16,11 +16,16 @@
  */
 
  import { BpmnProperty } from 'ama-sdk';
+import { BpmnElement } from '../../bpmn/elements';
+import { sanitizeString } from 'ama-sdk';
 
 const propertyKey = BpmnProperty.name;
 
 const get = element => element.businessObject[propertyKey];
 const set = (modeling: Bpmn.Modeling, element: Bpmn.DiagramElement, value: any) => {
+    if (element.type === BpmnElement.Process) {
```

Comment:
This is a hack, please create a custom processName.handler.ts, what is used for the name of the process. All the other cases can use the good old name.handler.ts